src/ChluIPFS.js

```diff
@@ -110,10 +135,12 @@ class ChluIPFS {
     }
 
     async readReviewRecord(multihash, options = {}) {
+        await this.waitUntilReady();
         return await this.reviewRecords.readReviewRecord(multihash, options);
     }
 
     async storeReviewRecord(reviewRecord, options = {}) {
+        await this.waitUntilReady();
```

Comment:
The error potentially thrown from https://github.com/ChluNetwork/chlu-ipfs-support/blob/74bac239a2adbd940c317694421d8885d811ec9d/src/ChluIPFS.js#L107 will have to be handled by the front end. Is that what we are implying here? Do we handle the error in chlu-demo then?

Reply:
Yes I think it should be handled in the frontend, we could also automatically start the node but I don't think we should risk the eventuality of having two nodes running in the same browser tab: they would step on each other unless they are correctly configured to avoid doing so

Reply:
👍 As long as the frontend handles the error, it is all good.